general/base_import/import_faq.rst

```diff
@@ -0,0 +1,201 @@
+=======================
+Import CSV file to Odoo
+=======================
+
+
+
+Frequently Asked Questions
+--------------------------
+
+
+Need to import data from an other application?
+----------------------------------------------
+
+In order to re-create relationships between different records, you should use the unique identifier from the original application and map it to the **ID** (External ID) column in Odoo. When you import an other record that links to the first one, use **XXX/ID** (XXX/External ID) to the original unique identifier.
```

Comment:
You are not going to like it but keep the 80 char limit for your text. It's way easier when we will need to correct typos or change a few things in a long text block.